backend/packages/worker/src/worker/parsing/models.py

```
@@ -1,4 +1,4 @@
-from typing import List, Optional
+from typing import List
```

Ruff (0.15.11)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)